org.bible.Rosary.yaml

```diff
@@ -0,0 +1,34 @@
+id: org.bible.Rosary
```

Comment:
Is this really the domain of the developer? 

> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

The checklist isn't a CAPTCHA.

Reply:
I have fixed this it is now proper.